bootstrap.js

```diff
@@ -7,54 +7,61 @@ const index = fs.readFileSync('index.html', 'utf8');
 const ResponseBuilder = require('./app/ResponseBuilder');
 
 module.exports = app => {
-    app.get('/*', (req, res) => {
+    app.all('/*', (req, res) => {
```

Comment:
Could we add some additional tests around PUT, POST methods to ensure this works correctly?

Reply:
This would make a lot of sense.

Side note; personally I was interested in the `OPTIONS` method, which I didn't even know existed until yesterday :).

Happened when trying to use `fetch()` from Javascript in Chrome through the proxy.